clowder-framework/clowder2-frontend
# Revisit update dataset #40

This PR works with backend PR: https://github.com/clowder-framework/clowder2-backend/pull/55
To test:
- npm run start:dev
- go to any of the dataset
- click "edit" on the name or description; edit
- see the change as well as the change in "modified" timestamp

<img width="366" alt="image" src="https://user-images.githubusercontent.com/13950475/160927744-9b947c44-08b9-47fa-b2d8-7fac00ea5d59.png">
